intermediate_source/scaled_dot_product_attention_tutorial.py

```diff
@@ -328,14 +323,82 @@ def generate_rand_batch(
 # the Shakespeare dataset.
 #
 
+######################################################################
+# Using SDPA with attn_bias subclasses`
+# ==========================================
+#
+# As of PyTorch 2.3, we have added a new submodule that contains tensor subclasses.
+# Designed to be used with ``torch.nn.functional.scaled_dot_product_attention``.
+# The module is named ``torch.nn.attention.bias`` and contains the following two
+# utilities for generating causal attention variants:
+#
+# - ``torch.nn.attention.bias.causal_upper_left``
+# - ``torch.nn.attention.bias.causal_lower_right``
+#
+# .. note:: 
+#    The current argument ``is_causal`` in ``torch.nn.functional.scaled_dot_product_attention``
+# is the same as using ``torch.nn.attention.bias.causal_upper_left``.
+#
+
+from torch.nn.attention.bias import causal_lower_right, causal_upper_left
```

Comment:
Why do I need both right away?

Reply:
I'd consider treating these one at a time

Reply:
I think the reason for doing both was IMO they make more sense in comparison to each other rather than in isolation